subgraphs/curve-finance/src/common/initializers.ts

```diff
@@ -388,6 +403,7 @@ export function getOrCreateLiquidityGauge(
 
     if (poolAddress) {
       const context = new DataSourceContext();
+      // eslint-disable-next-line rulesdir/no-string-literals
```

Comment:
Can remove as this rule was removed since you branched off master

Reply:
I will remove this line.